xous-rs/build.rs

```diff
@@ -1,3 +1,4 @@
+use core::panic;
```

Comment:
Just wondering -- was this line meant to be included in the patch? It's not clear to me how the proposed change requires the extra `use` statement.

Reply:
Sorry for that line, I added it accidentally when printing out the cargo envs to stdout so had to add a panic.